test/functional/comment_controller_test.rb

```diff
@@ -42,6 +42,29 @@ def teardown
     assert_template partial: 'questions/_comment'
   end
 
+  test 'should create wiki comments' do
+    UserSession.create(rusers(:bob))
+    assert_difference 'Comment.count' do
+      xhr :post, :create,
+          id: node(:wiki_page).nid,
+          body: 'Wiki comment'
+    end
+    assert_response :success
+    assert_not_nil :comment
+    #assert_template partial: 'wiki/_comment' 
+    #should uncomment above once comment displaying partial is implemented for wikis.
```

Comment:
Thanks @ananyo2012  and @jywarren , Yes i thought of it and we don't need a seperate table. With an additional field we could do the job. Thanks both of you!